### ISSUE TITLE
[4.x] Make laravel-azure-storage a suggested dependency 

### ISSUE THREAD

**Issue by Tofandel** [area17/twill#2570]
## Description
`matthewbdaly/laravel-azure-storage` is present in `require` of composer but not used anywhere from what I see

It also triggers a warning
Package matthewbdaly/laravel-azure-storage is abandoned, you should avoid using it. Use https://github.com/Azure-OSS/azure-storage-php instead.

**Comment by ifox** [comment]
Hi @Tofandel  this dependency is actually used when using `azure` in `MEDIA_LIBRARY_ENDPOINT_TYPE` and/or `FILE_LIBRARY_ENDPOINT_TYPE`.

Unfortunately it looks like efforts to maintain this integration for Laravel have stalled since Microsoft deprecated their SDK. 

We could remove the dependency and document installation or even install it dynamically if the azure type is used for now.

**Comment by Tofandel** [comment]
Then yes it should be in `suggest` dependencies and documented in the configuration of MEDIA_LIBRARY_ENDPOINT_TYPE and also using the new recommended lib now, removing it from dependencies now has to be done in v4 though (but changing it to a newer one can be done in v3)

**Comment by ifox** [comment]
I agree, but unfortunately the recommended library has no release yet.

**Comment by kerkness** [comment]
The suggestion provided during `composer update`  is  `Use azure-oss/storage-blob-laravel instead.`
